controllers/storageclient_controller.go

```diff
@@ -98,6 +101,7 @@ func (r *StorageClientReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Owns(&v1alpha1.StorageClaim{}).
 		Owns(&batchv1.CronJob{}).
 		Owns(&quotav1.ClusterResourceQuota{}, builder.WithPredicates(predicate.GenerationChangedPredicate{})).
+		Owns(&nbv1.NooBaa{}).
```

Comment:
Please add a generationChangedPredicate to this call, we are not interested in status or metadata changes.

Reply:
Added generationChangedPredicate call